### ISSUE TITLE
move gold standard data files to datasets/

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/erddap-gold-standard#64]
The gold standard example data files should be moved here under `datasets/` and added to datasets.xml

**Comment by srstsavage** [comment]
The three gold standard .nc datasets are already in the [./datasets](/ioos/erddap-gold-standard/tree/main/datasets) directory and referenced in [./erddap/content/datasets.xml](/ioos/erddap-gold-standard/tree/main/erddap/content/datasets.xml). Or did you mean something else?

**Comment by MathewBiddle** [comment]
We have some documentation on gold standard examples on the [metadata profile page](https://ioos.github.io/ioos-metadata/gold-standard-examples.html) which we would like to move over here. Plus ATN is nearly complete on their [file specification](https://ioos.github.io/ioos-atn-data/atn-sat-telem-specification-v1-0.html). Once that is ready, I'd like to include it in the `datasets/` directory or subdirectories therein.

**Comment by mwengren** [comment]
Yes, as @MathewBiddle mentioned, initially we put the doc page describing the gold standard datasets in the IOOS Metadata Profile repo (and I have no recollection of why... I think we planned to move them here eventually but it slipped through the cracks).  

Changes we need to make:

- [ ] migrate new 'gold standard' datasets to this repo (gliders, atn, etc)
- [ ] organize the /datasets directory here into data type categories (buoy, gliders, atn) and update datasets.xml config file accordingly
- [ ] [gold standard example datasets page](https://ioos.github.io/ioos-metadata/gold-standard-examples.html) should updated/improved to include new datasets and migrated to this repo
- [ ] links to that page should updated in [IOOS Metadata Profile page(s)](https://ioos.github.io/ioos-metadata/index.html) and other documentation repos
- [ ] update the live ERDDAP service at https://standards.sensors.ioos.us/erddap (and possibly change to just https://standards.ioos.us/erddap in the process?)
